docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source

```diff
@@ -0,0 +1,109 @@
+     * **Specify log and event sources**. Only specific sources will be selected if you go with this option.
+
+     For example, you select **Specific log and event sources**. In the next step, you will be able to view user activity, API usage in AWS services, IP traffic details, and route S3.
+
+  6. In **Log and event sources**, you see three options again:
+     * **CloudTrail**: View user activity and API usage in AWS services.
+     * **VPC flow logs**: View details about IP traffic to and from network interfaces in your VPC.
+     * **Route S3**
+
+     For example, you select Cloudtrail to get all the user activity and API usage in AWS services.
+
+     ![cloudtrail.png](/img/send-data/cloudtrail.png)
+
+  7. Access data.
+     * **S3** or **Lakeformation** is the storage option you can choose for how your subscribers will access the data.
+
+     :::note
+     Subscription URL will be added to the subscriber in the later step and is kept blank deliberately.
+     :::
+
+
+## Step 3. Create AWS Security Data Lake Source using Sumo Logic Environment
+
+When you create an AWS Security Data Lake source, you add it to a Hosted Collector. Before creating the source, identify the Hosted Collector you want to use from the existing ones or create a new Hosted Collector, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector.md).
+
+To create an AWS Security Data Lake source, follow the instructions below:
+1. In the Sumo Logic environment, go to **Manage Data** > **Collection** > **Collection**.
+2. On the **Collectors page, click **Add Source** next to a Hosted Collector.
+3. Select an existing Hosted Collector or the one you have just created.
+
+   ![aws-security-lake.png](img/send-data/aws-security-lake/png)
+
+4. Select **AWS Security Lake**.
+5. Enter a name for the new **Source**. A **Description** is optional.
+6. In the **Source Category**, enter any string to tag the output collected from this distinct source. (Category metadata is stored in a searchable field called    **_sourceCategory**).
+7. In **Fields**. Click the **+Add Field** link to add custom log metadata fields.
+8. Enter the required fields that you want to associate, each field needs a name (key) and value.
+  * ![green-circle.png](img/send-data/green-circle.png) A green circle with a checkmark shows up when a field exists and is enabled in the Fields table schema.
+  * ![orange-triangle.png](img/send-data/orange-triangle.png) An orange triangle with an exclamation point shows up when the field doesn't exist or is disabled in the **Fields table schema**.
+
+  :::important
+  In this case, an option to automatically add or enable the nonexistent fields to the **Fields table schema** is provided. If a field is sent to Sumo logic that does not exist in the **Fields table schema** or is disabled, it will be ignored and known as dropped field.
+  :::
+
```

Comment:
Fix indentation. Also, image links are broken; they don't exist at this file path. 

```suggestion
8. Enter the required fields that you want to associate, each field needs a name (key) and value.
   * ![green-circle.png](img/send-data/green-circle.png) A green circle with a checkmark shows up when a field exists and is enabled in the Fields table schema.
   * ![orange-triangle.png](img/send-data/orange-triangle.png) An orange triangle with an exclamation point shows up when the field doesn't exist or is disabled in the **Fields table schema**.
  :::important
  In this case, an option to automatically add or enable the nonexistent fields to the **Fields table schema** is provided. If a field is sent to Sumo logic that does not exist in the **Fields table schema** or is disabled, it will be ignored and known as dropped field.
  :::
```